src/commands/leaderboard.ts

```diff
@@ -32,10 +32,10 @@ const command: Command = {
 								index === 0
 									? "🥇"
 									: index === 1
-									? "🥈"
-									: index === 2
-									? "🥉"
-									: index + 1
+										? "🥈"
+										: index === 2
+											? "🥉"
+											: index + 1
```

Comment:
it's whatever but this is kinda of wild

Reply:
It's prettier, I find it stupid too but whatever